packages/web-app-files/src/mixins/actions/fetch.js

```diff
@@ -29,14 +37,15 @@ export default {
     }
   },
   methods: {
-    $_fetch_trigger(file, mimetype, isPublicFile) {
+    $_fetch_trigger({ resources, mimeType, isPublicFile }) {
       if (isPublicFile) {
-        const url = file.downloadURL
+        const url = resources[0].downloadURL
         window.open(url, '_blank')
         return
       }
 
-      const url = this.$client.helpers._webdavUrl + file.path
+      // FIXME: use presigned URL instead
```

Comment:
is this something for later or should happen in this pr?

Reply:
Not sure about the implications, @kulmann ?

Reply:
to get this merged it say keep the fixme comment here and do it later